src/main/java/org/iac2/entity/compliancejob/execution/ExecutionEntity.java

```diff
@@ -28,30 +43,42 @@ public class ExecutionEntity {
     @Temporal(TemporalType.TIMESTAMP)
     private Date endTime;
 
-    @Enumerated(EnumType.ORDINAL)
+    @Enumerated(EnumType.STRING)
     @NotNull
     private ExecutionStep currentStep;
 
-    @Enumerated(EnumType.ORDINAL)
+    @Enumerated(EnumType.STRING)
     @NotNull
     private ExecutionStatus status;
 
+    @Column(length = 10000)
+    private String instanceModel;
+
     private Boolean violationsDetected;
 
+    private String description;
+
     @ManyToOne
     @JoinColumn(name = "compliance_job_id", nullable = false)
     private ComplianceJobEntity complianceJob;
 
     @OneToMany(mappedBy = "execution")
-    private List<ComplianceIssueEntity> complianceIssueEntities;
+    @LazyCollection(LazyCollectionOption.FALSE)
+    private List<PluginUsageInstanceEntity> pluginUsageInstances;
 
+    @OneToMany(mappedBy = "execution")
+    @LazyCollection(LazyCollectionOption.FALSE)
```

Comment:
Well. the tests were not running properly (loading the "one" side of the "OneToMany" relation in a different session as the lazily loaded "many" side throws an exception). Now I learned that such methods need to be labeled as `@Transactional`.
Anyway, let's keep the approach you mentioned in-mind in case we might need it.